p2p/src/consensus.rs

```diff
@@ -9,7 +9,9 @@ pub(crate) fn consensus_encode_with_size<W: Write + ?Sized>(
 }
 
 pub(crate) fn parse_failed_error(msg: &'static str) -> bitcoin::consensus::encode::Error {
-    bitcoin::consensus::encode::Error::Parse(bitcoin::consensus::encode::ParseError::ParseFailed(msg))
+    bitcoin::consensus::encode::Error::Parse(bitcoin::consensus::encode::ParseError::ParseFailed(
+        msg,
+    ))
```

Comment:
ooph, that one is ugly. Perhaps we want to add a local import of `encode`. I'm not going to bother doing it though.